extruct/tool.py

```diff
@@ -25,6 +29,15 @@ def main(args=None):
         help='List of syntaxes to extract. Valid values any or all (default):'
              'microdata, opengraph, microformat json-ld, rdfa.'
              'Example: --syntaxes microdata opengraph json-ld')
+    arg('--uniform', default=False, 
+        help='''If True uniform output format of all syntaxes to a list of dicts.
+                Returned dicts structure:
+                {'@context': 'http://example.com', 
+                 '@type': 'example_type',
+                 /* All other the properties in keys here */
+                 }''')
+    arg('--schema_context', default='http://schema.org', 
+        help="schema's context for current page")
     args = parser.parse_args(args)
     metadata = metadata_from_url(args.url, args.syntaxes)
```

Comment:
It seems that arguments added above (uniform and schema_context) are not passed into ``metadata_from_url``